.github/actions/publish-versioned-docs/action.yml

```diff
@@ -0,0 +1,68 @@
+name: 'Publish Versioned Documentation'
+description: 'Publishes versioned documentation using Mike for MkDocs'
+inputs:
+  tag:
+    description: 'Version tag to publish (e.g., 1.0.0, 1.1.0)'
+    required: true
+  branch:
+    description: 'Branch to get documentation content from (defaults to current branch)'
+    required: false
+    default: ''
+  make_latest:
+    description: 'Make this version the latest/default'
+    required: false
+    default: 'true'
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.x
+        cache: pip
+
+    - name: Install dependencies
+      shell: bash
+      run: pip install -r requirements.txt
+
+    - name: Configure git for Mike
+      shell: bash
+      run: |
+        git config user.name "github-actions[bot]"
+        git config user.email "github-actions[bot]@users.noreply.github.com"
```

Comment:
can you use the `github.actor` variable to have the actual user you clicked the release button or pushed the tag ?

Reply:
Updated in https://github.com/scality/mountpoint-s3-csi-driver/pull/126/commits/ecec4a1d9b2514c144b58226c61fef4ae9ca0c42 for main and hotfix